dev/updgrade-calcite-parser

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
```

Comment:
Nit, looks like there's a typo in the script filename here :).

Reply:
it was also typoed in the pom.xml :D fixed that too :)
thank you :)